includes/definitions/apc-epdu.yaml

```diff
@@ -0,0 +1,12 @@
+os: apc-epdu
```

Comment:
Is it worthwhile to have a different OS?

Reply:
maybe i missed something but i think apc os use completely different values in the same mib. My PDUs have no info for the oids used by apc os.